pdc_client/runner.py

```diff
@@ -49,10 +66,22 @@ def __init__(self):
         self.logger = logging.getLogger('pdc')
 
     def load_plugins(self):
+        ns, _ = self.parser.parse_known_args()
+        server = ns.__getattribute__('server')
+        if not server:
+            raise TypeError('Server must be specified')
+        config = pdc_client.read_config_file(server)
+        if config:
+            plugins = config[CONFIG_PLUGINS_KEY_NAME]
+            if not plugins:
+                plugins = DEFAULT_PLUGINS
+        else:
+            plugins = DEFAULT_PLUGINS
+
         for dir in PLUGIN_DIRS:
             self.logger.debug('Loading plugins from {0}'.format(dir))
             for name in os.listdir(dir):
-                if not name.endswith('.py'):
+                if not name.endswith('.py') or name not in plugins:
```

Comment:
It's a minor thing. If use `in` here, it's a litter better to set  `plugins` as a set instead of a list. `in` operation for a set is O(1) complexity (https://wiki.python.org/moin/TimeComplexity). But not big deal for this small list.